app/src/confirm_email.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { useParams } from "react-router-dom";
+
+async function confirmEmail(credentials) {
+  return fetch("http://localhost:1000/confirm_email/<token>", {
+    method: "POST",
+    credentials: "include",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(credentials),
+  }).then((response) => response.json());
+}
+
+function ConfirmEmail() {
+  const params = useParams();
+  const email_token = params["*"];
```

Comment:
```suggestion
  const emailToken = params["*"];
```
We should use camelCase in JS